src/test/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFsUnitTest.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hellbender.tools.walkers;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class CombineGVCFsUnitTest {
+
+    @DataProvider(name="breakIntermediateStopSites")
+    public Object[][] getIntermediateStopSitesData() {
+        return new Object[][] {
+                { new SimpleInterval("contig", 1, 1), 1, Collections.EMPTY_LIST },
+                { new SimpleInterval("contig", 1, 2), 1, Arrays.asList(1) },
+                { new SimpleInterval("contig", 1, 2), 2, Arrays.asList(1) },
+                { new SimpleInterval("contig", 1, 10), 2, Arrays.asList(1, 3, 5, 7, 9) },
+                { new SimpleInterval("contig", 1, 10), 5, Arrays.asList(4, 9) },
+                { new SimpleInterval("contig", 1, 100), 25, Arrays.asList(24, 49, 74, 99) },
+
+                { new SimpleInterval("contig", 10, 10), 2, Arrays.asList(9) },
+                { new SimpleInterval("contig", 10, 10), 5, Arrays.asList(9) },
+
+                // TODO ?? is this the correct result ?
+                { new SimpleInterval("contig", 10, 10), 10, Arrays.asList(9) },
```

Comment:
ehhh... i don't consider this critical or even important at all, but all of these tests that return Collections.EMPTY_LIST as a result are only ever failing on the first breakpoint interval, which technically goes through a new codepath. If you wanted you could add a test { new SimpleInterval("contig", 110, 120), 100, Collections.EMPTY_LIST }, to be more through.

Reply:
Agreed on the first comment. I view the purpose of this test as validating that we don't create invalid intermediate intervals at this stage of the calculation, since thats what was originally failing - I think the fact that they get filtered out later is ok. Ideally we'd have additional unit tests for the subsequent parts of the calculation, but they require much more state.

I think the additional test you mentioned should return {"99"}, not the empty list, for this method. Again, that stop site will get filtered later as its outside of the interval. Its testing the case where the band multiple is smaller than the start of the close interval, which is essentially the same as the `{ new SimpleInterval("contig", 10, 20), 5, Arrays.asList(9, 14, 19) }` test.  Doesn't hurt to add it though.